pom.xml

```diff
@@ -66,8 +66,8 @@
         <jts.bundle.version>1.12_1</jts.bundle.version>
         <karaf.version>2.4.3</karaf.version>
         <lux.version>1.2</lux.version>
-        <opengis.bundle.version>${org.geotools.version}_1</opengis.bundle.version>
-        <org.geotools.bundle.version>${org.geotools.version}_1</org.geotools.bundle.version>
+        <opengis.bundle.version>8.4_1</opengis.bundle.version>
```

Comment:
Do we need to update org.geotools.version to something newer, rather than hardcoding the version here?